content/10.diagnostics.md

```diff
@@ -168,7 +155,53 @@ Sensitivity and specificity metrics were obtained by comparison with results fro
+Outside of COVID-19 diagnostics, the applications of LFTs are broad; they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine [@DOI:10.1007/s00216-008-2287-2].
+
+A recent review surveyed the performance of LFTs for detection of current SARS-CoV-2 infection [@PMID:34407759].
+This review covered 24 studies that included more than 26,000 total LFTs.
+They found significant heterogeneity in test sensitivities ranging from 37.7% (95% CI 30.6-45.5) to 99.2% (95% CI 95.5-99.9).
+The specificities of these tests were more homogeneous, spanning 92.4% (95% CI 87.5-95.5) to 100.0% (95% CI 99.7-100.0).
+
+Despite having lower sensitivity than PCR tests, LFTs occupy an important niche in the management of SARS-CoV-2.
+Current infection detection by LFTs enables the scale and speed of testing that is beneficial to managing viral spread.
+LFTs were made available freely to citizens in the United Kingdom at least until December 2021 [@DOI:10.1136/bmj.n1760] and to citizens of the United States in early 2022 [@url:https://www.whitehouse.gov/briefing-room/statements-releases/2022/01/14/fact-sheet-the-biden-administration-to-begin-distributing-at-home-rapid-covid-19-tests-to-americans-for-free].
+These tests are particularly useful for ruling out SARS-CoV-2 infection in cases where the likelihood of infection is low (e.g., asymptomatic individuals) and positives (including false positives) can be validated with testing by alternate means [@doi:10/hk34].
+
+##### Enzyme-Linked Immunosorbent Assay
+
+Enzyme-linked immunosorbent assay (ELISA) is a very sensitive immunoassay that can be considered a gold standard for the detection of biological targets, including antibodies and antigenic proteins [@pubmed:32310382].
+It can be used to generate either quantitative or qualitative results that can be returned within a few hours [@doi:10.1007/s00253-020-11061-5].
+ELISA builds on the idea that antibodies and antigens bind together to form complexes [@pubmed:32310382], utilizing enzymes that react with an antigen as reporter labels [@doi:10.1373/clinchem.2005.051532].
+The reagents are immobilized on a plate and anything in the sample that binds is retained [@url:https://www.bosterbio.com/protocol-and-troubleshooting/elisa-principle].
+The exact strategy for tagging with a reporter enzyme varies among different types of ELISA [@url:https://www.bosterbio.com/protocol-and-troubleshooting/elisa-principle; @pubmed:32310382].
+For COVID-19 diagnostics, ELISAs have been designed to detect all or part of the antigenic Spike protein [@doi:10.12688/wellcomeopenres.15927.1].
+
+One of these assays used two monoclonal antibodies specific to the nucleocapsid of SARS-CoV-2 to evalaute the relationship between the effect of (estimated) viral load on the ability of the assay to detect the SARS-CoV-2 antigen [@doi;10.1080/14787210.2021.1976144].
```

Comment:
```suggestion
One of these assays uses two monoclonal antibodies specific to the nucleocapsid of SARS-CoV-2 to evalaute the relationship between the effect of (estimated) viral load on the ability of the assay to detect the SARS-CoV-2 antigen [@doi;10.1080/14787210.2021.1976144].
```

Reply:
I had made a comment here regarding serological testing, but you covered it in the next PR :)